### PR TITLE
cursor: fix focus.followMouse glitches

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: src/cursor.c [hunk 1]
```diff
@@ -508,6 +508,19 @@ cursor_update_focus(struct server *server)
...
 	struct cursor_context ctx = get_cursor_context(server);
+
+	if (ctx.view && rc.focus_follow_mouse) {
+		desktop_focus_and_activate_view(&server->seat, ctx.view);
+		if (rc.raise_on_focus) {
+			/*
+			 * Call view method directly as desktop_move_to_front()
+			 * contains a call to cursor_update_focus() and thus
+			 * loops inifinitely
+			 */
```
Comment (Consolatis) [inline src/cursor.c:519]: Heh, nice catch :)
As said, it was only compile tested.

**Comment by Consolatis** [comment]
@jech another round of testing would be greatly appreciated.

**Comment by Consolatis** [comment]
Alright then :)

**Comment by jech** [comment]
It seems to work fine for switching desktops and closing the topmost window, thanks to both of you.

Testing together with https://github.com/labwc/labwc/pull/829, I still need action "Focus" after I do action "Lower".  This is consistent with Openbox, but not what I'd expect from FFM.  (I can live with either behaviour, it's just that I find the current behaviour counter-intuitive; so I guess a decision from a wise person is needed.)

**Pull request merged**